docs/source/cuda/cuda_array_interface.rst

```diff
@@ -513,6 +513,14 @@ The following Python libraries have adopted the CUDA Array Interface:
 - `ArrayViews <https://github.com/xnd-project/arrayviews>`_
 - `JAX <https://jax.readthedocs.io/en/latest/index.html>`_
 - `PyCUDA <https://documen.tician.de/pycuda/tutorial.html#interoperability-with-other-libraries-using-the-cuda-array-interface>`_
+- `DALI: the NVIDIA Data Loading Library <https://github.com/NVIDIA/DALI>`_ :
+
+    - `TensorGPU objects
+      <https://docs.nvidia.com/deeplearning/dali/master-user-guide/docs/data_types.html#nvidia.dali.backend.TensorGPU>`_
+      expose the CUDA Array Interface.
+    - `The External Source operator
+      <https://docs.nvidia.com/deeplearning/dali/master-user-guide/docs/supported_ops.html#nvidia.dali.fn.external_source>`_
+      consumes objects exporting the CUDA Array Interface.
```

Comment:
These links point to doc builds from HEAD of mainline? Current releases are `s/master-user-guide/user-guide/`, any view on which is appropriate?

Reply:
Yes, it is the latest successful nightly build from TOT, replacing them with:
https://docs.nvidia.com/deeplearning/dali/user-guide/docs/supported_ops.html#nvidia.dali.fn.external_source
and
https://docs.nvidia.com/deeplearning/dali/user-guide/docs/data_types.html#nvidia.dali.backend.TensorGPU

Will give you the latest release. @stuartarchibald which ones do you prefer? 

The update about CAI in TensorGPU (only the documentation, the support I think was present for about a year) will be available in DALI 1.2 which is scheduled for the release around the end of the month.

Reply:
From a stability point of view I'd go with that which is released, unless you regularly encourage users to use development builds? @gmarkall as code owner do you have a view?

Reply:
@stuartarchibald that is a good point, fixing.